modules/acquia_cms_headless/composer.json

```diff
@@ -69,7 +69,8 @@
                 "3111456 - Unable to resolve path on node in other language than default": "https://www.drupal.org/files/issues/2022-11-30/decouple_router-3111456-resolve-language-issue-58.patch"
             },
             "drupal/subrequests": {
-                "Get same results on different request": "https://www.drupal.org/files/issues/2019-07-18/change_request_type-63049395-09.patch"
+                "3049395 - Page Cache causes different subrequests to return the same responses": "https://www.drupal.org/files/issues/2019-07-18/change_request_type-63049395-09.patch",
+                "3338312 - Incompatible with latest versions of symfony/http-foundation": "https://git.drupalcode.org/project/subrequests/-/merge_requests/14.patch"
```

Comment:
Cross posting this from the earlier MR because this is critical and needs to be addressed before merging this MR: 

> We shouldn't use MR directly as a patch because anyone could push commits to the MR and it would be automatically loaded when running composer install.

https://github.com/acquia/acquia_cms/pull/1499#discussion_r1151618555

Reply:
done